docs/pages/setup/reference/config.mdx

```diff
@@ -344,8 +344,6 @@ auth_service:
             # app_id should always also be listed as a facet
             - https://localhost:3080
             - https://localhost
-            - localhost:3080
```

Comment:
fyi non https localhost is fine, it's a special case.

Reply:
Good call, added back with a comment.